SimplenoteWidgets/Tools/BuildConfiguration.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+enum BuildConfiguration: String {
+    case debug
+    case `internal`
+    case appStore
+    case release
+    case unknown
+
+    static var current: BuildConfiguration {
+
+        #if DEBUG
+            return .debug
+        #elseif BUILD_INTERNAL
+            return .internal
+        #elseif BUILD_APP_STORE
+            return .appStore
+        #elseif BUILD_RELEASE
+            return .release
+        #else
+
+            return .unknown
+        #endif
+    }
+
+    static func `is`(_ configuration: BuildConfiguration) -> Bool {
+        return configuration == current
+    }
+}
+
+extension BuildConfiguration: CustomStringConvertible {
+    var description: String {
+        return self.rawValue
+    }
+}
```

Comment:
This file is duplicated (Original is in `Simplenote/BuildConfiguration`). Can we drop this copy and update the Target Membership of the former file?

![Uploading Screen Shot 2021-07-26 at 11.24.54 PM.png…]()

Reply:
I had done that originally, but Simplenote/BuildConfiguration imports Tracks, which we had previously decided we did not want to include in the extensions.  I can have it import Tracks and not use it if that would be preferred.  I was trying to avoid bundling the extra package if it wasn't needed

Reply:
Tracks appears not to be used anywhere in `BuildConfiguration`.

No idea exactly why there's an `import AutomatticTracks` in there, but you can just remove that import, and update the target membership all over 😄